apps/system_message/resource.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+
+
+class SystemMessagesResource(Resource):
+    schema = {
+        "is_active": {"type": "boolean", "default": False},
+        "type": {
+            "type": "string",
+            "allowed": ["warning", "alert", "primary/info", "success"],
+            "required": True,
+        },
+        "message_title": {"type": "string", "required": True},
+        "message": {"type": "string", "required": True},
+        "user": {"type": "string", "required": True},
+    }
+
+    resource_methods = ["GET", "POST"]
+    item_methods = ["GET", "PATCH", "DELETE"]
+    no_privileges = True
```

Comment:
wouldn't it be easier to just define a new privilege?

Reply:
only the "administrator" users should be able to perform create, update and delete operations, you think we should define a new privilege?

Reply:
yep I just wonder why we handle this differently than other authorizations